cirq/study/resolver.py

```diff
@@ -181,6 +181,20 @@ def value_of(
             self._deep_eval_map[value] = self.value_of(v, recursive)
         return self._deep_eval_map[value]
 
+    def resolved_to_single_step(self) -> 'ParamResolver':
```

Comment:
Do you have other uses for this method? If not, I'd just inline it on line 195 below.

Reply:
Done.